spec/workflow/task/retry_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+module Kuroko2::Workflow::Task
+  describe Retry do
+
+    subject(:task) { Retry.new(node, token) }
+    let(:node) { Kuroko2::Workflow::ScriptParser.new(token.script).parse.find(token.path).next }
+    let(:definition) { create(:job_definition) }
+    let(:instance) { create(:job_instance, job_definition: definition) }
+    let(:token) do
+      Kuroko2::Token.create(uuid: SecureRandom.uuid, job_definition: definition, job_instance: instance, script: script)
+    end
+
+    describe '#validate' do
+      context 'with valid script format' do
+        let(:script) do
+          <<-EOF.strip_heredoc
+            retry: count=3 sleep_time=10
+              noop:
+              noop:
+            noop:
+          EOF
+        end
+
+        it 'passes validation' do
+          expect{ task.validate }.not_to raise_error
```

Comment:
[MUST] Put a whitespace before { of brace blocks.

https://github.com/cookpad/styleguide/blob/master/ruby.en.md#method-calls